css-layout-classes/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="styles.css">
+    <title>
+      CSS Layout Classes
+    </title>
+  </head>
+  <body>
+    <div class="container">
+
+      <h1>Choose your character!</h1>
+
+      <div class="row col-one-third">
+        <img src="images/link.png" alt="link">
+        <div>
+          <h2>Super Smash Bros. Ultimate</h2>
+          <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Corporis numquam sequi, aut pariatur neque veniam sint omnis possimus deserunt! Est et aut, hic reiciendis iure at. Architecto quas ipsam quisquam.</p>
+        </div>
+        <img src="images/yoshi.png" alt="yoshi">
+      </div>
+
+      <div class="row">
+        <div col-one-fourth>
```

Comment:
What is this `col-one-fourth`? I can assure you that HTML has no idea what it is. It should be a class.